docs/build/interoperate-qiskit-qasm2.mdx

```diff
@@ -51,63 +59,179 @@ import qiskit.qasm2
 circuit = qiskit.qasm2.load("myfile.qasm")
 ```
 
+
 <span id="custom-instructions"></span>
-## Custom quantum instructions
+### Link OpenQASM 2 gates with Qiskit gates
+
+By default, Qiskit's OpenQASM 2 importer treats the include file `"qelib1.inc"` as a *de facto* standard library.
+The importer treats this file as containing precisely the gates it is described to contain in [the original paper defining OpenQASM 2](https://arxiv.org/abs/1707.03429).
+Qiskit will use the built-in gates in [the circuit library](../api/qiskit/circuit_library) to represent the gates in `"qelib1.inc"`.
+Gates defined in the program by manual OpenQASM 2 `gate` statements will, by default, be constructed as custom [Qiskit `Gate` subclasses](../api/qiskit/qiskit.circuit.Gate).
+
+You can tell the importer to use specific [`Gate`](../api/qiskit/qiskit.circuit.Gate) classes for the given `gate` statements it encounters.
+You can also use this mechanism to treat additional gate names as "built-in", that is, not requiring an explicit definition.
+Qiskit will typically be more efficient if you can tell it which gate classes it should use for `gate` statements outside of `"qelib1.inc"`.
```

Comment:
```suggestion
For efficiency, specify which gate classes to use for `gate` statements outside of `"qelib1.inc"`.
```

Reply:
I'm not convinced I've come up with the best sentence here 🤣 but my aim is to take out the anthropomorphism and simplify to a statement of more direct action. Maybe a pair of fresh eyes can help -- will ask around

Reply:
Haha, no trouble - I have a pretty large habit of anthropomorphising code.  My natural release notes would probably say things like "`qasm2.loads` learned a new keyword argument" and the like - not exactly formal terminology!

Fwiw, does something like:
```suggestion
If you specify which gate classes to use for `gate` statements outside of `"qelib1.inc"`, the resulting circuit will typically be more efficient to work with.
```
help?  I think I've got an awkward word/clause order in that, but just throwing it into the hopper for other ways of saying it.

Reply:
Yep that sounds better, thanks!